[Internal] Usage: Add README.md for usage samples (#4069)
Azure/azure-cosmos-dotnet-v3: Microsoft.Azure.Cosmos.Samples/Usage/AzureFunctions/README.md

# Cosmos client with Azure Function

THis sample demonstrates how to maintain a Cosmos client and reuse the instance among Azure Function executions.

## Prerequisites

- Azure Cosmos DB NoSQL Account

>   - Create a DataBase and Container

Reviewer: ```suggestion
- Create a Database and Container
```

## Run

> Before running the application you need fill out `EndPointUrl` and `AuthorizationKey` params in the [AppSettings.json](AppSettings.json)

Reviewer: ```suggestion
Before running the application you need fill out `EndpointUrl` and `AuthorizationKey` params in the [AppSettings.json](AppSettings.json)
```

# Cosmos client with Azure Function

> This sample demonstrates how to maintain a Cosmos client and reuse the instance among Azure Function executions.

Reviewer 2: ```suggestion
This sample demonstrates how to maintain a Cosmos client instance following Singleton pattern and reuse the instance across Azure Function executions.
```

## Prerequisites

- Azure Cosmos DB NoSQL Account
- Create a Database and Container
- Microsoft.Azure.Cosmos NuGet [package](http://www.nuget.org/packages/Microsoft.Azure.Cosmos/)

## Run

> Before running the application you need fill out `EndpointUrl` and `AuthorizationKey` params in the [AppSettings.json](AppSettings.json)

Reviewer 2: ```suggestion
Before running the application you need fill out `EndpointUrl` and `AuthorizationKey` params in the [AppSettings.json](../appSettings.json)
```
